extensions/hibernate-orm/deployment/src/main/java/io/quarkus/hibernate/orm/deployment/HibernateOrmConfigPersistenceUnit.java

```diff
@@ -381,4 +395,30 @@ public boolean isAnyPropertySet() {
         @ConfigItem
         public OptionalLong objectCount;
     }
+
+    @ConfigGroup
+    public static class HibernateOrmConfigPersistenceUnitFetch {
+        /**
+         * The size of the batches used when loading entities and collections.
+         *
+         * `-1` means batch loading is disabled. This is the default.
+         *
+         * @asciidoclet
+         */
+        @ConfigItem(defaultValue = "-1")
+        public int batchFetchSize;
+
+        /**
+         * The maximum depth of outer join fetch tree for single-ended associations (one-to-one, many-to-one).
+         *
+         * A `0` disables default outer join fetching.
```

Comment:
Detail: This Javadoc comment should have Quarkus' `@asciidoclet` tag, since it uses Asciidoc markup here for the `0` literal. I know this was copy-pasted from my own contribution (sorry about that!).

Reply:
Done.

Reply:
> Detail: This Javadoc comment should have Quarkus' `@asciidoclet` tag, since it uses Asciidoc markup here for the `0` literal. I know this was copy-pasted from my own contribution (sorry about that!).

I believe the `0` literal should be handled without the need for the `@asciidoclet` tag. The tag is to be used in cases where we want the javadoc comments to be written as-is and not parsed ( e.g considered as valid asciidoc input that make use of various asciidoc stylings etc ) e.g https://github.com/quarkusio/quarkus/blob/499992e3b73cedc14a668646fb4966f1b3502701/extensions/hibernate-orm/deployment/src/main/java/io/quarkus/hibernate/orm/deployment/HibernateOrmConfigPersistenceUnit.java#L38-L71.

Reply:
Unfortunately I am not that familiar with asciidoc. Do you think I should remove the `@asciidoc` tags as literals should be parsed without it?

Reply:
Yes, removing the `@asciidoclet` should have no effect. But before editing it, let's wait for @Sanne and @gsmet full functional / configuration review first. This way we'll avoid round trips.

Reply:
I suppose if `@asciidoclet` is removed the  `` `0` `` should be changed to `{@code 0}`.